src/components/CommentForm.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import "./CommentForm.css";
+import { postComment } from "../api";
+
+const CommentForm = ({ article_id, onCommentSubmit }) => {
+  const [body, setBody] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    const commentData = {
+      username: "cooljmessy", // will add user context when other core work complete
+      body: body,
+    };
```

Comment:
A hardcoded user is fine - but this ideally would be in a user context!
As a stretch goal, a user selection page would be a nice optional extra